kubescape.yaml

```diff
@@ -15,6 +15,7 @@ environment:
       - cmake
       - git
       - go
+      - gobump
```

Comment:
fwiw we don't need to explicitly add gobump here because the bobump pipeline referenced below will include it https://github.com/wolfi-dev/os/blob/cafd7ffd4451b7f0f6f7e0450538691fdbd38813/pipelines/go/bump.yaml#L7

Not a big issue, just figured it best to mention.

Reply:
Sorry, my bad forgot to remove it.